### PR TITLE
Invite sign up E2E tests

### PR DESCRIPTION
**Pull request by paolodamico**
## Changes

This PR adds some tests related to the critical flow of accepting an invite (as discussed on Slack ~a week ago). The following test cases are covered:
- Creating an invite
- Authenticated user cannot use invite for someone else
- New user can use an invite
- ~Existing user can use an invite intended for them. EDIT: Test is too flaky, cannot be retried. Removing for now.~

In addition, the tests detected a bug that made it impossible to leave an organization which is fixed in this PR. The page was reloaded too soon for the request to be completed.

## Checklist

- [X] N/A, All querysets/queries filter by Organization, by Team, and by User
- [X] N/A. Django backend tests
- [X] N/A. Jest frontend tests
- [X] Cypress end-to-end tests

### CODE REVIEW

File: frontend/src/scenes/organization/TeamMembers/membersLogic.tsx [hunk 1]
```diff
@@ -57,6 +60,11 @@ export const membersLogic = kea<membersLogicType>({
             }
             actions.loadMembers()
         },
+        postRemoveMember: async ({ memberId }) => {
+            if (memberId === userLogic.values.user?.id) {
+                location.reload()
+            }
+        },
```
Comment (Twixes) [inline frontend/src/scenes/organization/TeamMembers/membersLogic.tsx:67]: Why so?

File: frontend/src/scenes/organization/TeamMembers/Invites.tsx [hunk 2]
```diff
@@ -98,6 +98,7 @@ function _Invites(): JSX.Element {
                 pagination={false}
                 loading={invitesLoading}
                 style={{ marginTop: '1rem' }}
+                className="invites-table"
```
Comment (Twixes) [inline frontend/src/scenes/organization/TeamMembers/Invites.tsx:101]: If this is not for styling, then `data-attr` is usually the way.

File: frontend/src/scenes/organization/TeamMembers/Members.tsx [hunk 3]
```diff
@@ -241,6 +238,7 @@ export function Members({ user }: { user: UserType }): JSX.Element {
                 pagination={false}
                 style={{ marginTop: '1rem' }}
                 loading={membersLoading}
+                className="org-members-table"
```
Comment (Twixes) [inline frontend/src/scenes/organization/TeamMembers/Members.tsx:241]: Same as `className="invites-table"`.

File: frontend/src/scenes/organization/TeamMembers/membersLogic.tsx (hunk 1 above)
Comment (Twixes) [inline frontend/src/scenes/organization/TeamMembers/membersLogic.tsx:67]: Why so?

**Comment by paolodamico** [comment]
Could use a bit of help here, tests run totally fine locally, not sure why they fail in the CI

**Comment by paolodamico** [comment]
Moving this to draft until we get #3701 in.

**Comment by Twixes** [comment]
The other PR is in, not sure to what degree this particular PR makes sense at the moment?

**Comment by paolodamico** [comment]
Yeah, I will make the relevant adjustments and get this ready.

**Comment by paolodamico** [comment]
Cleaned up and ready for a quick review!

**Review by Twixes — APPROVED** [review]
Nice

**Pull request merged**